codemp/client/cl_uiapi.cpp

```diff
@@ -290,6 +290,7 @@ static qboolean CL_G2API_GetBoltMatrix_NoRecNoRot( void *ghoul2, const int model
 		return qfalse;
 	}
 
+	re->G2API_BoltMatrixReconstruction( qfalse );
```

Comment:
I don't think this should be here... I think it might only in cgame because of... https://github.com/JACoders/OpenJK/blob/master/codemp/cgame/cg_players.c#L9252-L9258

Reply:
It is how the original SDK behaves I believe. I was just trying to make it all in sync with original behavior.

Reply:
Please check with that